org.racket_lang.Racket.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">org.racket_lang.Racket</id>
```

Comment:
Given DrRacket is the entrypoint, shouldn't it be the last fragment of ID?

Reply:
The desktop launcher is also DrRacket, so that likely should be the app name, too.